src/core/messages.ts

```diff
@@ -259,8 +259,6 @@ export interface Messages extends EmitsDiscontinuities {
    *
    * Should you wish to restore a deleted message, and providing you have the appropriate permissions,
    * you can simply send an update to the original message.
-   * Note: This is subject to change in future versions, whereby a new permissions model will be introduced
-   * and a deleted message may not be restorable in this way.
```

Comment:
Why remove this? Isn't this a helpful disclaimer?

Reply:
I've added this back in, I saw this as a reference to conditional updates, but it could still be useful to note regardless of that use-case.